scala3doc/src/dotty/dokka/tasty/comments/MemberLookup.scala

```diff
@@ -103,7 +112,12 @@ trait MemberLookup {
       def hackResolveModule(s: Symbol): Symbol =
         if s.flags.is(Flags.Module) then s.moduleClass else s
 
-      val matched = syms.find(matches)
+      // val syms0 = syms.toList
+      // val matched0 = syms0.find(matches)
+      // if matched0.isEmpty then
```

Comment:
Debugging leftover?

Reply:
I intentionally left it commented out, it's going to be useful when debugging again.